garak/resources/tap/generator_utils.py

```diff
@@ -4,7 +4,7 @@
 from typing import Union
 
 from garak.generators.openai import chat_models, OpenAIGenerator
-from garak.generators.huggingface import Model
+from garak.generators.huggingface import ConversationalPipeline
 
```

Comment:
Presumably yes! We'd just need to validate that the model is conversational. I only recommended these model types because they're named in the paper but presumably we could use any kind of generator if we wanted to.